dandi/metadata.py

```diff
@@ -147,7 +151,7 @@ def timedelta2duration(delta):
         sec = delta.seconds
         if delta.microseconds:
             # Don't add when microseconds is 0, so that sec will be an int then
-            sec += delta.microseconds / 1000000
+            sec += delta.microseconds / 1e6
```

Comment:
do you really want to change this?

Reply:
yes... why are you asking?
do you have a reason to keep it? I could revert this change if there is any reason, but can't think of any, so you have to tell me

Reply:
do you prefer 10**6?

Reply:
what you have is fine :)